src/components/layout/layout-v2.tsx

```diff
@@ -35,68 +41,32 @@ const Main = styled.main`
    */
   display: grid;
   grid-template-columns:
-    1fr
-    min(821px, calc(100% - 2 * 24px))
-    1fr;
+    [main-start] minmax(24px, 1fr)
+    [content-start] minmax(0, 820px) [content-end]
+    minmax(24px, 1fr) [main-end];
 
   > * {
-    grid-column: 2;
+    grid-column: content;
   }
 
   /** make sure the distance to the footer is always the same */
-  padding-bottom: 121px;
+  padding-bottom: 120px;
 
   ${up('md')} {
     padding-bottom: 160px;
   }
 `;
 
-export const FullWidth = styled.div`
-  width: 100%;
-  grid-column: 1 / 4;
-`;
-
 const HeaderStickyContainer = styled.div`
   position: fixed;
   top: 0;
   width: 100%;
   z-index: 100;
 `;
 
-const HeroContainer = styled.div`
-  position: relative;
-  width: 100%;
-  height: 520px;
-
-  ${up('md')} {
-    height: 640px;
-  }
-`;
-
 const BreadcrumbContainer = styled.div<{ hero: boolean }>`
   margin: ${(props) => !props.hero && `calc(${HEADER_HEIGHT} + 16px)`} 24px 16px;
 `;
-const useAnchorTagScrolling = (): void => {
```

Comment:
separate file because it's cluttering the view and deserves its own file for sure